source/manual/test-and-build-a-project-with-github-actions.html.md

```diff
@@ -0,0 +1,266 @@
+---
+owner_slack: "#govuk-developers"
+title: Test & build a project with GitHub Actions
+section: Testing
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-05-11
+review_in: 6 months
+---
+
+[GitHub Actions](https://github.com/features/actions) is an automated workflow
+system that GOV.UK uses for [Continuous Integration (CI)][ci], we also have
+[Jenkins](/manual/testing-projects.html) and [Concourse](/manual/concourse.html)
+that provide similar functionality. In [GOV.UK RFC 123][] we decided that
+GitHub Actions is the preferred platform for GOV.UK CI usage where the wider
+platform integration of Jenkins is not required.
+
+Please note that we cannot currently make use of the [Secrets][actions-secrets]
+in GitHub Action workflows, there are on-going discussions to resolve this. In
+the meantime please **do not add any secrets to workflows**. For further
+details of this restriction or please contact
+[GOV.UK senior tech](mailto:govuk-senior-tech@digital.cabinet-office.gov.uk).
+
+Deciding the CI platform for a GOV.UK project can be based on the
+following considerations:
+
+- As Jenkins is hosted within GDS' network it can be used to trigger other
+  internal systems to perform jobs during or following a build (for example,
+  triggering [publishing end-to-end tests][e2e-tests] or triggering a
+  deployment)
+- Jenkins can be used for any services that require authentication to external
+  systems (for example, [releasing a gem
+  automatically][gem-release] during a master build), since secrets are not
+  permitted within GitHub Actions
+- As GitHub Actions are not constrained by the [GOV.UK Puppet
```

Comment:
Do we need to say this? I've assumed that, since we have an accepted RFC, we only need to cover reasons to postpone using GitHub Actions?

Reply:
Yeah, could probably turn this into a list of restrictions rather than a more comparison style of list. 

My concern though is that might open a different can of worms which leads to a doc on choosing your CI platform.

Reply:
So my understanding is that: all CI should be GitHub actions, and we're working on a couple of issues with the approach, and any affected repos should be excluded _for now_. 

Hopefully that avoids any implication of choice. Does that help?